services/storage/src/simcore_service_storage/modules/celery/__init__.py

```diff
@@ -17,6 +18,7 @@ async def on_startup() -> None:
         assert celery_settings  # nosec
         celery_app = create_app(celery_settings)
         app.state.celery_client = CeleryTaskQueueClient(celery_app)
+        set_fastapi_app(celery_app, app)
```

Comment:
not sure this makes sense.
here you are in the manager setting up the celery client. why not just pass the redis client as a field of the CeleryTaskQueueClient instead?